src/kw_manager.sh

```diff
@@ -265,15 +282,245 @@ function rename_kw_manager_group()
   fi
 }
 
+# This function add a new contact to a given kw mail group
+#
+# @contacts_string: The string with all the contacts informations
+# @group_name: The name of the group which the contacts will be added.
+#
+# Returns:
+# returns 0 if successful, non-zero otherwise
+function add_kw_manager_contacts()
+{
+  local contacts_list="$1"
+  local group_name="$2"
+  local group_id
+  declare -A _contacts_array
+  declare -A _new_contacts_array
+
+  if [[ -z "$contacts_list" ]]; then
+    complain 'The contacts list is empty'
+    return 22 #EINVAL
+  fi
+
+  check_existent_group "$group_name" && exit_msg 'Error while checking group existence'
```

Comment:
Are you sure you want to exit here? Maybe you want to return and let the main function dealing with the exit.